e2etest/bgptests/bgp.go

```diff
@@ -84,7 +89,8 @@ var _ = ginkgo.Describe("BGP", func() {
 
 	ginkgo.AfterEach(func() {
 		if ginkgo.CurrentGinkgoTestDescription().Failed {
-			dumpBGPInfo(cs, f)
+			dumpBGPInfo(ReportPath, ginkgo.CurrentGinkgoTestDescription().TestText, cs, f)
+			Reporter.Dump(10*time.Minute, strings.Replace(ginkgo.CurrentGinkgoTestDescription().TestText, " ", "-", -1))
```

Comment:
Should we extract the values: 10*time.Minute and strings.Replace(.....) to variables
with a meaningful name?
Asking because: 
1. We reuse the values in multiple places, so it will be easier to change if we'll ever need to.
2. The purpose of those values can't trivially be inferred from the function name Dump,
so meaningful variable names could spare the reader from looking in the Dump function definition.